quad.go

```diff
@@ -0,0 +1,159 @@
+// Copyright ©2015 The gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package quad provides numeric integration of single variable functions by quadrature.
+//
+package quad
```

Comment:
I wrote that before noticing that this package is intended only for 1d integration, in that case `quad` is also fine for me.